SwiftScripts/.swiftpm/xcode/xcshareddata/xcschemes/SwiftScripts-Package.xcscheme

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does this file need to be committed?

Reply:
It's going to be added everytime we open the project in Xcode 14. Doesn't matter. Probably easier to keep it than to `.gitignore` it.